Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-SD-3-B.yml

```diff
@@ -16,23 +16,24 @@ id:
 
 #The title of this activity
 title:
-  Files and repositories are checked periodically for secrets that should be protected.
+  Favor run-time exposure to secrets and improve lifecycle management.
 
 #Describe the benefit that is achieved by implementing this activity
 benefit: 
-  Risk of leaking production secrets is mitigated by removing all manual interactions and regular regeneration.
+  Risk of leaking production secrets is reduced by limiting access only during run-time and by regenerating regularly.
 
 #A one sentence description of the activity
 shortDescription:
-  Regenerate secrets dynamically during every deployment using well-hardened process wherever possible. 
+  Use run-time solutions to minimize access to application secrets. Properly manage the secret lifecycle.
 
 #A multi-paragraph description of the activity
 longDescription: 
-  Where secrets are not predefined or dependant on another system, generate them during the deployment process. Follow appropriate best practices such as using a cryptographically secure pseudorandom number generator if you generate this value randomly. Alert any manual access to secrets in the production environment. 
+  Whereever possible, rely on run-time techniques to manage access to the secrets required to execute the application. This is typically achieved by means of a vaulting solution with an quary interface that can be used by the application based on a bootstrapping secret.
+  
+  Implement proper lifecycle management for production secrets, and force the generation of new secrets as much as possible, and for every application instance. 
```

Comment:
"proper" sounds subjective

Reply:
Changed into appropriate. Not sure whether this is better though.

Reply:
What about implement lifecycle management for production secrets (ensure at least x and y)?

Reply:
OK. done.